### PR TITLE
[WIP] prow job for building containerd

### PR DESCRIPTION
**Pull request by Barraud-L**

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/164#" title="Author self-approved">Barraud-L</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
Welcome @Barraud-L! It looks like this is your first PR to ppc64le-cloud/test-infra 🎉

**Comment by ltccci** [comment]
Hi @Barraud-L. Thanks for your PR.

I'm waiting for a [ppc64le-cloud](https://github.com/orgs/ppc64le-cloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/ppc64le-cloud/test-infra/blob/master/README.md) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: config/jobs/ppc64le-cloud/builds/containerd-postsubmit.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+postsubmits:
+  containerd/containerd:
```
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/builds/containerd-postsubmit.yaml:2]: Heads up: This doesn't work unless we add a webhook to the `containerd/containerd` repository.

Comment (mkumatag) [reply config/jobs/ppc64le-cloud/builds/containerd-postsubmit.yaml:2]: alternate way would be to use the periodic type of job where is trigger based on the interval(like cronjob)

**Comment by ltccci** [comment]
@Barraud-L: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pull-test-infra-prow-checkconfig | abf114f928692aeed93a8fe8b73a6e2b8db1b7f4 | [link](https://prow.ppc64le-cloud.org/view/s3/prow-logs/pr-logs/pull/ppc64le-cloud_test-infra/164/pull-test-infra-prow-checkconfig/1376781730450509824) | `/test pull-test-infra-prow-checkconfig`

[Full PR test history](https://prow.ppc64le-cloud.org/pr-history?org=ppc64le-cloud&repo=test-infra&pr=164). [Your PR dashboard](https://prow.ppc64le-cloud.org/pr?query=is:pr+state:open+author:Barraud-L).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by Rajalakshmi-Girish** [comment]
As per request from @florencepascual closing this PR.

**Pull request closed without merging**